lib/src/qr_code_scanner.dart

```diff
@@ -65,7 +65,7 @@ class _QRViewState extends State<QRView> {
   void initState() {
     super.initState();
     _observer = LifecycleEventHandler(resumeCallBack: updateDimensions);
-    WidgetsBinding.instance!.addObserver(_observer);
+    WidgetsBinding.instance.addObserver(_observer);
```

Comment:
Fatal errors can be caused non-nullable widget binding instance due to lower dart version
I think you should also change the dart version range from
https://github.com/juliuscanute/qr_code_scanner/blob/f55ba22b33ab9d0d88c71968d3ba6eaed8f048eb/pubspec.yaml#L8
to
``` yaml
  sdk: '>=2.17.0 <3.0.0'
```

Reply:
change the flutter version too.

```flutter: ">=3.0.0"```

Reply:
> change the flutter version too.
> 
> `flutter: ">=3.0.0"`

@X-SLAYER Is this changed with this pull request? I didn't find any changes on yaml file. Moreover, ignore my suggestion on sdk version change. Changing Flutter version is more accurate.